app/modules/dollarable.rb

```diff
@@ -0,0 +1,9 @@
+module Dollarable 
```

Comment:
Stealing this over to our side...

Reply:
Let me know if I am wrong, but I think this method will drop trailing 0's. Do you check for 8.20 instead of 8.2 anywhere?

Reply:
@LawrenceWhalen hmm I took that to be view logic so I do it in the view with `number_to_currency`

Reply:
Does that seem right to you? ^

Reply:
I should probably condense my testing here anyway, though, since I refactored into this, the tests are still living elsewhere

Reply:
Reviewed and confirmed that my view logic deals with this issue on the show page. I also updated my edit form, though, so thanks for noting this. I still believe that dealing with trailing 0s belongs in the view still, so I am not going to update that in the module itself.

Reply:
@LawrenceWhalen sorry meant to tag you there ^